canonical-web-and-design/ubuntu.com
# Change meta titles on managed apps and kafka page

## Done

- Changed the meta titles for /managed and /managed/kafka to match the [issue](https://github.com/canonical-web-and-design/web-squad/issues/3031)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/managed
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Ensure that the new meta title is in effect
- Navigate to http://0.0.0.0:8001/managed/kafka and do the same
